ci-operator/step-registry/hypershift/mce/agent/oadp/v2/hypershift-mce-agent-oadp-v2-commands.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+  source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export KUBECONFIG="${SHARED_DIR}/nested_kubeconfig"
+oc create namespace oadp-helper
+IMAGE=$(oc get clusterversion version -ojsonpath='{.status.desired.image}')
+TOOLS_IMAGE=$(oc adm release info ${IMAGE} --image-for=tools)
+oc create secret generic oadp-kubeconfig-secret --from-file=kubeconfig="$KUBECONFIG" -n oadp-helper
+cat <<EOF | oc apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: oadp-helper
+  namespace: oadp-helper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: oadp-helper
+  template:
+    metadata:
+      labels:
+        app: oadp-helper
+    spec:
+      containers:
+        - name: patch-container
+          image: ${TOOLS_IMAGE}
+          command: [ "sh", "-c", "while true; do sleep 3600; done" ]
+          volumeMounts:
+            - name: kubeconfig
+              mountPath: /etc/kubernetes
+              readOnly: true
+      volumes:
+        - name: kubeconfig
+          secret:
+            secretName: oadp-kubeconfig-secret
+
+EOF
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+cat <<EOF > /tmp/miniocred
+[default]
+aws_access_key_id=admin
+aws_secret_access_key=admin123
+EOF
+
+oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=/tmp/miniocred
+
+cat <<EOF | oc apply -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: dpa-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: minio
+          s3ForcePathStyle: 'true'
+          s3Url: 'http://virthost.ostest.test.metalkube.org:9000'
+          insecureSkipTLSVerify: "true"
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: oadp-backup
+          prefix: hcp
+        provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - kubevirt
+        - csi
+      customPlugins:
+        - name: hypershift-oadp-plugin
+          image: quay.io/hypershift/hypershift-oadp-plugin:latest
```

Comment:
Hey @LiangquanLi930 this image is deprecated I will delete to avoid confusions. 
These are the ones generated in Konflux:
- quay.io/redhat-user-workloads/crt-redhat-acm-tenant/hypershift-oadp-plugin-oadp-1-5:oadp-1.5 --> Release OADP-1.5
- quay.io/redhat-user-workloads/crt-redhat-acm-tenant/hypershift-oadp-plugin-main:latest --> Main